code/modules/reagents/chemistry/reagents/other_reagents.dm

```diff
@@ -2126,9 +2128,6 @@
 	description = "A colorless liquid that suppresses violence in its subjects. Cheaper to synthesize than normal Pax, but wears off faster."
 	metabolization_rate = 1.5 * REAGENTS_METABOLISM
 
-/datum/reagent/peaceborg
-	can_synth = FALSE
-
```

Comment:
This change is not mentioned, but this type path isn't used anywhere, and all of its child reagents set `can_synth` to TRUE, so it can be safely removed?